### ISSUE TITLE
 the accuracy of In-Shop Clothes Retrieval model

### ISSUE THREAD

**Issue by RenLuXi** [open-mmlab/mmfashion#22]
Hello
  Thank you for your great work!
  When I run ` python3 demo/test_retriever.py ` and load the model using:

> --config   configs/retriever/roi_retriever_vgg.py
> --input  demo/retrieve_demo1.jpg
> --checkpoint   checkpoint/Retrieve/vgg/epoch_190.pth

  And got:

> pretrained model checkpoint/vgg16.pth
> The model and loaded state dict do not match exactly
> 
> unexpected key in source state_dict: classifier.0.weight, classifier.0.bias, classifier.3.weight, classifier.3.bias, classifier.6.weight, classifier.6.bias

  Retrived top5 results are very inaccurate, no picture is right. Is there something wrong with my way of using it? If so, how to choose args?I also want to know how to get the matching degree of each picture？
  The gallery_embeds time is so long ，It's about four minutes.
  Forgive me for being stupid.Could you please give me some suggestions.
  Thank you very much

**Comment by veralauee** [comment]
Because you just used the vgg16.pth pretrained via ImageNet. You should load our pretrained checkpoint so that the accuracy can be correct.

**Comment by liuziwei7** [comment]
Will close this issue for now. Please feel free to re-open it in case of any further questions.

**Comment by huytuong010101** [comment]
@RenLuXi  have you fixed it?
